components/FilterForm/FilteredCarsList.js

```diff
@@ -19,13 +20,15 @@ function FilteredCarsList({ list }) {
         <StyledList>
           {list.map((car) => (
             <li key={car.id}>
-              <Image
-                style={{ borderRadius: "10%" }}
-                src={car.imageSource}
-                alt={car.model}
-                width={300}
-                height={150}
-              />
+              <Link href={`/detailsCar/${car.id}`}>
+                <Image
+                  style={{ borderRadius: "10%" }}
```

Comment:
There seems to be a mix of styled components and inline styles in your project. Is there a reason for this?

If you want to use styled components on the Image component from nextjs i think you can do something like this. 

``` js
const StyledImage = styled(Image)`
  border-radius: 10%;
`;
```

Reply:
Thank you very much!
I really appreciate your work by reviewing my PR. I have fixed comments and will work more on styling after functionality is ready. Thanks again!